docs/content/3.providers/cloudimage.md

```diff
@@ -47,10 +45,31 @@ export default defineNuxtConfig({
 })
 ```
 
+### `apiVersion`
+
+- Type: **String**
+- Default: `v7`
+
+Allow to use a specific version of API.
+
+**Note:** For tokens created after **20.10.2021**, `apiVersion` need to be set to empty string or you can use `cdnURL` with value `https://{token}.cloudimg.io`.
+
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  image: {
+    cloudimage: {
+      token: 'demo',
+      apiVersion: 'v7' // optional
```

Comment:
if this is the default for new tokens we should probably add it above to the example (set to an empty string)

Reply:
With the breaking changes, I've added a bit more explanation to docs. Should be much clearer how Cloudimage integration works.